src/styles/blocks/about.scss

```
@@ -0,0 +1,49 @@
+.about-us {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  flex-wrap: wrap;
+
+  &__title {
+    font-size: 48px;
+    line-height: 48px;
+    font-weight: 700;
+    letter-spacing: -1.5px;
+    color: #000;
+
+    &-filter {
+      color: #0075ff;
+    }
+  }
+
+  &__article {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    flex-wrap: wrap;
+
+    &-subtitle {
+      padding-bottom: 16px;
+      font-size: 24px;
+      line-height: 30px;
+      font-weight: 700;
+      color: #0075ff;
+    }
+
+    &-text {
+      padding-bottom: 16px;
+      max-width: 500px;
+      font-size: 16px;
+      line-height: 27px;
+      font-weight: 400;
+      color: #3e3e3e;
+
+      &-bold {
```

```suggestion
      &-bold {
```
If this is a modifier let's use `_` or `--`

---

I used it --, but linter swears